tests/e2e/test_api.py

```diff
@@ -185,6 +185,56 @@ def test_query_call_with_improper_payload():
         assert "missing" in response.text
 
 
+def test_valid_question_improper_conversation_id(response_eval) -> None:
+    """Check the REST API /v1/query with POST HTTP method for improper conversation ID."""
+    endpoint = "/v1/query"
+    eval_query, _ = get_eval_question_answer(response_eval, "eval1", "with_rag")
+
+    with metrics_utils.RestAPICallCounterChecker(
+        metrics_client, endpoint, status_code=requests.codes.internal_server_error
+    ):
+        response = client.post(
+            endpoint,
+            json={"conversation_id": "not-uuid", "query": eval_query},
+            timeout=LLM_REST_API_TIMEOUT,
+        )
+        assert response.status_code == requests.codes.internal_server_error
+        json_response = response.json()
+        expected_response = {
+            "detail": {
+                "response": "Error retrieving conversation history",
+                "cause": "Invalid conversation ID not-uuid",
+            }
+        }
+        assert json_response == expected_response
+
+
+@pytest.mark.rag
```

Comment:
why is this marked w/ rag?  it doesn't seem dependent on rag

Reply:
ok I changed the param for response checker and removed the mark. Let's see on CI

Reply:
we are green @bparees